globals.go

```diff
@@ -278,6 +278,7 @@ const (
 	DefaultJasperPort = 2385
 
 	GlobalGitHubTokenExpansion = "global_github_oauth_token"
+	GithubAppPrivateKey        = "github_app_private_key"
```

Comment:
Does this need to be exported to other packages if we're only using it in the `evergreen` package?

Reply:
privatized